src/applications/third-party-apps/components/SearchResult/index.jsx

```diff
@@ -0,0 +1,147 @@
+// Node modules.
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import join from 'lodash/join';
+import map from 'lodash/map';
+
+export class SearchResult extends Component {
+  static propTypes = {
+    item: PropTypes.shape({
+      appURL: PropTypes.string.isRequired,
+      categories: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+      description: PropTypes.string.isRequired,
+      iconURL: PropTypes.string.isRequired,
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+      permissions: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+      platforms: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+      privacyPolicyURL: PropTypes.string.isRequired,
+      termsOfServiceURL: PropTypes.string.isRequired,
+    }).isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      show: false,
+    };
+  }
+
+  setShow = show => {
+    this.setState({ show });
+  };
+
+  render() {
+    const { item } = this.props;
+    const { setShow } = this;
+    const { show } = this.state;
+
+    return (
+      <li className="third-party-app vads-u-display--flex vads-u-flex-direction--column vads-u-margin-bottom--2 vads-u-padding--3 vads-u-border-color--gray-lightest vads-u-border--2px">
+        <div className="vads-u-display--flex vads-u-align-items--center vads-u-justify-content--space-between">
+          {/* App Icon */}
+          <img
+            alt={item?.name}
```

Comment:
If `item` is null, we should probably just exit render early because otherwise our component will be just a bunch of empty placeholders. Can we remove the `?.` operators in exchange for an early return `if (!item)`? It's just confusing to me otherwise if we're constantly expecting `item` to be null